modules/apps/callbacks/callbacks_test.go

```diff
@@ -263,23 +294,21 @@ func (s *CallbacksTestSuite) AssertHasExecutedExpectedCallbackWithFee(
 	s.AssertHasExecutedExpectedCallback(callbackType, isSuccessful)
 }
 
-// OverrideSendMsgWithAssertion overrides both chains' SendMsgsOverride function to assert whether the
-// transaction is successful or not.
-func OverrideSendMsgWithAssertion(chain *ibctesting.TestChain, expPass bool) {
+// overrideSendMsg overrides both chains' SendMsgs function to a version that doesn't require
+// that the transaction is successful.
+func overrideSendMsg(chain *ibctesting.TestChain) {
```

Comment:
This is needed because v7.3's `SendMsg` requires the tx to be successful, so we override it. (chain.go is not one of the copied files)